function-reference/install-functions/text.xml

```diff
@@ -431,6 +431,8 @@ The <c>*into</c> functions create the directory if it does not already exist.
       Normally executed to exclude from stripping.
       Eg. <c>dostrip -x /path/to/important.so</c>.  May also be used to include binaries
       to strip when <c>RESTRICT=strip</c> without the -x option.
+      Note that provided paths must be absolute relative to <c>${ED}</c>, i.e., as if
+      <c>${ED}</c> was the filesystem root.
```

Comment:
Two small points:
- "Absolute relative" is an oxymoron :smile:. It really is a path _relative_ to `${ED}`.
- I think we don't put commas after `i.e.` because the devmanual is in British English.
```suggestion
      Note that provided paths must be relative to <c>${ED}</c>, i.e. as if
      <c>${ED}</c> was the filesystem root.
```

Maybe also change the example above to something other than `/path/to/important.so` which isn't very helpful. Something starting with `/usr` or `/opt` might be better there.

Reply:
I don't think it's an oxymoron because in this case `dostrip -x /usr/lib/node_modules` works, but `dostrip -x usr/lib/node_modules` does not.  I would consider the second a "relative path".  Is there some sort of better phrasing you can think of to call the former?

Reply:
How about the following wording for the whole item?
```
      Controls stripping of executables. Normally used to exclude from
      stripping, e.g. <c>dostrip -x /usr/$(get_libdir)/important.so</c>.
      Without the <c>-x</c> option, may also be used to include binaries to
      strip when <c>RESTRICT=strip</c>. Note that <c>${ED}</c> is prepended
      to provided paths, i.e. as if <c>${ED}</c> was the filesystem root.
```

Reply:
Assuming I understand this correctly, then what about adding the follow to what ulm suggests:

> This means that the path argument of dostrip must start with a leading slash ('/'), but the path itself is relativ to ED.

Reply:
That's not correct either, as it works without the leading slash (as specified by the PMS). It is implemented like this in Portage (in phase-helpers.sh):
```
			[[ ${f:0:1} = / ]] || f="/${f}"
```

Reply:
Sounds good, went with this phrasing.